sql/017-operators-ore.sql

```diff
@@ -401,24 +401,31 @@ CREATE OPERATOR <=(
 
 -----------------------------------------------------------------------------------------
 
-
 DROP FUNCTION IF EXISTS cs_encrypted_ore_64_8_compare(a cs_encrypted_v1, b cs_encrypted_v1);
 
 CREATE FUNCTION cs_encrypted_ore_64_8_compare(a cs_encrypted_v1, b cs_encrypted_v1)
   RETURNS integer AS $$
+   DECLARE
+    a_ore bytea[];
+    b_ore bytea[];
   BEGIN
-    RETURN compare_ore_64_8_v1(cs_ore_64_8_v1(a), cs_ore_64_8_v1(b));
+    a_ore = ARRAY[(a->>'o')::bytea];
+    b_ore = ARRAY[(b->>'o')::bytea];
+
+    RETURN compare_ore_64_8_v1(a_ore, b_ore);
```

Comment:
Explicitly using `bytea[]`, but need new function definition for these types too.